chain-extensions/types/xvm/src/lib.rs

```diff
@@ -65,4 +64,6 @@ pub struct XvmCallArgs {
     pub to: Vec<u8>,
     /// Encoded call params
     pub input: Vec<u8>,
+    /// Value to transfer
+    pub value: Balance,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `XvmCallArgs` struct has been expanded with a new `value` field of type `Balance`. This field is likely meant to specify the amount to transfer during the cross-VM call. Verify that this new field is correctly utilized in the codebase and appropriately handled during the cross-VM call process.
```diff
     pub to: Vec<u8>,
     /// Encoded call params
     pub input: Vec<u8>,
+    /// Value to transfer
+    pub value: Balance,
 }
```
LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->